mobile/makefile

```diff
@@ -1,3 +1,5 @@
+.PHONY: build watch create_app_icon create_splash build_release_android
```

Comment:
Do you mean to include this change in this PR?

Reply:
Hi, yes, the makefile didn't do anything when I ran build as it thought no files changed

.PHONY marks the commands after it as unstable (always run) [doc](https://www.gnu.org/s/make/manual/html_node/Phony-Targets.html)

> A phony target is one that is not really the name of a file; rather it is just a name for a recipe to be executed when you make an explicit request. There are two reasons to use a phony target: to avoid a conflict with a file of the same name, and to improve performance.

I'm happy to remove it though

Reply:
No, that should be fine, thanks